packages/bpk-component-breakpoint/src/useMediaQuery-test.tsx

```diff
@@ -66,10 +66,8 @@ describe('useMediaQuery', () => {
     }));
 
     const view = renderHook(() => useMediaQuery('(min-width: 768px)'));
-
-    expect(view.result.all.length).toBe(1);
-
-    expect(view.result.all[0]).toBe(true);
+    
+    expect(view.result.current).toBe(true);
```

Comment:
The structure of `renderHook` returns a different object where the value is located